Sources/PointFreeTestSupport/PointFreeTestSupport.swift

```diff
@@ -0,0 +1,111 @@
+import Foundation
```

Comment:
I did something I wanted to do at kickstarter too... move all the mocks to its own package so that we can use them in tests _and_ playgrounds!